src/frontend/src/pages/HomePage/WorkPackagesByTimelineStatus.tsx

```diff
@@ -61,24 +94,41 @@ const WorkPackagesByTimelineStatus: React.FC = () => {
             <Card
               variant="outlined"
               key={wbsPipe(wp.wbsNum)}
-              sx={{ minWidth: 'fit-content', mr: 3, background: theme.palette.background.default }}
+              sx={{ minWidth: 'fit-content', mr: 3, background: theme.palette.background.default, borderRadius: '16px' }}
             >
-              <CardContent sx={{ padding: 3 }}>
-                <Link
-                  variant="h6"
-                  component={RouterLink}
-                  to={`${routes.PROJECTS}/${wbsPipe(wp.wbsNum)}`}
-                  sx={{ marginBottom: 2 }}
-                >
-                  {wbsPipe(wp.wbsNum)} - {wp.name}
-                </Link>
-                <DetailDisplay label="End Date" content={datePipe(wp.endDate)} paddingRight={2} />
-                <DetailDisplay label="Progress" content={percentPipe(wp.progress)} paddingRight={2} />
-                <DetailDisplay label="Engineering Lead" content={fullNamePipe(wp.projectLead)} paddingRight={2} />
-                <DetailDisplay label="Project Manager" content={fullNamePipe(wp.projectManager)} paddingRight={2} />
-                <Typography>
+              <CardContent sx={{ padding: 2 }}>
+                <Grid container spacing={2}>
+                  <Grid item xs={9}>
+                    <Typography variant="h6" sx={{ textDecoration: 'underline', align: 'right' }}>
+                      {wp.wbsNum.projectNumber} - {wp.projectName}
+                    </Typography>
+                    <Link
+                      variant="h5"
+                      component={RouterLink}
+                      to={`${routes.PROJECTS}/${wbsPipe(wp.wbsNum)}`}
+                      sx={{ marginBottom: 2 }}
+                      noWrap={true}
+                    >
+                      {wbsPipe(wp.wbsNum)} - {wp.name}
+                    </Link>
+                    <Typography sx={{ marginTop: 1 }}>
```

Comment:
these all should be individual grid items, thats why the small one is all messed up.